### PR TITLE
View list and detail applications for organisers.

### PR DESCRIPTION
**Pull request by phalt**
Redacted

### CODE REVIEW

**Comment by phalt** [comment]
This Pull Request adds the following features:

- View for seeing all applications submitted to a particular page/city/event (Partial for #18)
- View for seeing the detail view of application questions and answers (Partial for #19)
- A decorator for views that allows only super user and members of the page/city/event team to view. Returns a 404 Not Found if you do not meet these criteria.

There is no sorting for applications or the ability to triage them yet. I'm absolutely terrible with front end pretty things, so I've just written the basic html tag stuff. I hope this helps!

File: applications/utils.py [hunk 1]
```diff
@@ -27,13 +27,11 @@
         "title": "Which operating system do you use?",
         "question_type": "choices",
         "choices": "Mac OS X; Windows; Linux",
-        "has_option_other": True,
```
Comment (phalt) [inline applications/utils.py]: These were breaking after being removed in a previous migration :)

Comment (andyosuna) [inline applications/utils.py]: I commented those lines out and am working like that xP

**Comment by olasitarska** [comment]
This looks amazing @phalt! Obviously some css definitely missing from the templates, but we can take care of that in another issue :) 

Thank you, you rock! :+1: :tulip: :koala:

**Pull request merged**